core/app/views/refinery/admin/_menu.html.erb

```diff
@@ -1,4 +1,25 @@
-<nav id="menu">
-  <%= render partial: "refinery/admin/menu_item",
-             collection: current_refinery_user.active_plugins.in_menu -%>
-</nav>
+<button type="button" class="btn btn-primary sidebar-left-opener fixed-t-l z-5 hidden-sm-up"><i class="icon icon-bars"></i></button>
```

Comment:
An Admin UI who meets WCAG's requirements is important in Quebec. Refinery CMS could be considered to be used in government applications. At this time, it can't. This is a code refactor, so we could do the best practice now but you're right for the fact : it will make more sense only if whole refinery is using that.